src/components/AddRepoForm/AddRepoForm.js

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from 'react';
+import { useSelector, useDispatch } from 'react-redux'
+import { getRepository } from '../../actions'
+import { Form, InputGroup, FormControl, Button } from 'react-bootstrap';
+import './AddRepoForm.scss'
+
+const AddRepoForm = () => {
+  const [repoName, setRepoName] = useState('')
+  const error = useSelector(state => state.repositories.error)
```

Comment:
It's better to use selectors instead of direct accessing state https://redux.js.org/recipes/computing-derived-data